README.asciidoc

```diff
@@ -131,6 +134,13 @@ The plugin defines the following extension properties in the `docker` closure:
 |`apiVersion`    |String    |null                     |The remote API version. For most cases this can be left null.
 |=======
 
+[WARNING]
+====
+Default value of `docker.url` property is deprecated.
+It will be replaced to `unix:///var/run/docker.sock` in upcoming releases.
+Please adjust your build scripts accordingly.
```

Comment:
I think a more sensible default, at least for now, would be for `tcp://localhost:2375`. The purpose of using the remote API is to connect to engines regardless of their OS. Defaulting to unix assumes one is using a unix based OS and thus connecting to unix only docker-engine. TCP url on the other hand is more generic.